Jenkinsfile

```diff
@@ -490,6 +499,150 @@ az aks create                      \
+                                            // variables to reference a user in AWS with the correct privileges to create an EKS
+                                            // cluster: https://github.com/weaveworks/eksctl/issues/204#issuecomment-450280786
+                                            container('eksctl') {
+                                                sh "eksctl create cluster --name ${clusterName} --region ${awsRegion} --zones ${awsZones.join(',')} --version ${kversion} --node-type m5.large --nodes 3 --timeout 60m --kubeconfig \$KUBECONFIG --tags 'platform=${platform},branch=${BRANCH_NAME},build=${BUILD_URL}'"
+                                                waitForRollout("kube-system", 30)
+                                            }
+
+                                            writeFile([file: 'kubeprod-manifest.jsonnet', text: """
+(import "manifests/platforms/eks.jsonnet") {
+  config:: import "${clusterName}-autogen.json",
+  letsencrypt_environment: "staging",
+  prometheus+: import "tests/testdata/prometheus-crashloop-alerts.jsonnet",
+}
+"""
+                                            ])
+                                        }{
+                                            // update glue records in parent zone
+                                            def nameServers = []
+                                            container('aws') {
+                                                def output = sh(returnStdout: true, script: """
+id=\$(aws route53 list-hosted-zones-by-name --dns-name "${dnsZone}" --max-items 1 --query 'HostedZones[0].Id' --output text)
+aws route53 get-hosted-zone --id "\$id" --query DelegationSet.NameServers
+"""
+                                                )
+                                                nameServers = readJSON(text: output)
+                                            }
+                                            insertGlueRecords(dnsPrefix, nameServers, "60", parentZone, parentZoneResourceGroup)
+                                        }
+                                    } finally {
+                                        // Only if there is a valid Kubernetes configuration
+                                        if (fileExists("${env.WORKSPACE}/.kubeconf")) {
+                                            // Uninstall BKPR
+                                            withEnv([
+                                                "PATH+KUBECFG=${tool 'kubecfg'}",
+                                                "PATH+KUBECTL=${tool 'kubectl'}",
+                                                "KUBECONFIG=${env.WORKSPACE}/.kubeconf",
+                                            ]) {
+                                                // This is required for AWS in order to release/destroy ELB network interfaces.
+                                                sh """
+                                                    #!/bin/bash
```

Comment:
is this shebang really needed? and also the following `set` commands. I believe `set -x` is the default

Reply:
It is not needed here (afaics), and we should revert to standard `sh` (without the shebang, and without explicit `set +x`/`set -x` (?!))

Reply:
Fixed.